README.md

```diff
@@ -81,12 +81,17 @@ There will be four meetings per year.   Three half day meetings (3 hours) will b
 
 **Solid Operations Advisory Group (formerly Solid team)** 
 
-This group advises on key areas including the solid website, hosting structure for PODs,  solidcommunity.net, github.
+In contrast, the Solid Operations Advisory Group is a volunteer group that is self-nominated and elected by the community. This group is not expected to do substantive day-to-day work, but rather act as representatives that guide key decisions - for instance, they would have been consulted on whether to proceed with the recent solidcommunity.net migration. They would also be advising on the infrastructure roadmap which would guide where employed maintenance staff in the Project/Initiative Team would focus their time.
```

Comment:
You would need to reference Technology Team to draw the contrast, or drop the word "contrast". Rest of the text is OK!

Reply:
Good catch, resolved in https://github.com/solid/odi-governance/pull/16/commits/04e27bd6df290dfcfd0476d7f9e7ab25c3c476b8